src/app.js

```diff
@@ -62,6 +62,7 @@ var accountTradeHandler = function (username, password, sharedSecret) {
     });
 
     client.on("loggedOn", function () {
+        this.setPersona(SteamUser.EPersonaState.Online) // we want to appear Online 24/7
```

Comment:
And it would be better to add config option where user can select if he want to appear online instead of forcing that behaviour.